.github/workflows/ci.yaml

```diff
@@ -32,6 +32,27 @@ jobs:
         with:
           config: ./typos.toml
 
+  docker-build:
```

Comment:
Add CI to run docker build, does make sense? i can update it to master if necessarily.